content/articles/spring-boot-caching/index.md

```diff
@@ -0,0 +1,332 @@
+Cache reduces the number of database reads resulting in reduced cost of operation and improved application speed. Spring Boot provides an abstraction layer for cache implementation. 
```

Comment:
Don't directly jump into the topic.

Give a brief history before cache, and after cache. State how it's helpful or state how it has affected applications.

State why we use springboot here. How is better than others?